azurerm/internal/services/resource/feature_resource_test.go

```diff
@@ -0,0 +1,104 @@
+package resource_test
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance/check"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/resource/parse"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/tf/pluginsdk"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+type FeatureResource struct {
+}
+
+func TestAccFeatureResource_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_feature", "test")
+	r := FeatureResource{}
+
+	data.DataSourceTest(t, []acceptance.TestStep{
+		{
+			Config: r.basic(),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+	})
+}
+
+func TestAccFeatureResource_requiresImport(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_feature", "test")
+	r := FeatureResource{}
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.requiresImportBasic(),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.RequiresImportErrorStep(r.requiresImport),
+	})
+}
```

Comment:
Hi @ms-henglu - just noticed while passing through that these will need to be skipped (via `t.Skip()`) until out acceptance test setup can support having features toggled without potentially causing issue elsewhere in the testing suite.

Reply:
Thanks! @jackofallops , I added skip on these tests.